lesson_24/script2.js

```diff
@@ -0,0 +1,113 @@
+let number1 = prompt('Please, enter first numbers');
+let number2 = prompt('Please, enter second numbers');
+
+function sum(a, b) {
+  if ((a === 0 || a) && (b === 0 || b)) {

```

Comment:
I know that you took this code from lesson, but now I think such checking will be better:

`typeof a === 'number' && !isNaN(a)`